### PR TITLE
Feature/v4 user endpoint php update list multipart statustransition

### PR DESCRIPTION
**Pull request by Rrathinasabapath**
Added changes for User enpoint for:
*Update
*List
*Multipart upload
*User status transition

### CODE REVIEW

File: tests/Hyperwallet/Tests/HyperwalletTest.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
 class HyperwalletTest extends \PHPUnit_Framework_TestCase {
 
+    private $includeIntegrationTest = false; //change this value to true if integration tests have to be run
```
Comment (ramahalingam) [inline tests/Hyperwallet/Tests/HyperwalletTest.php:37]: Pls remove this unused variable.

Comment (Rrathinasabapath) [reply tests/Hyperwallet/Tests/HyperwalletTest.php]: Removed the unused line

**Review by ramahalingam — CHANGES_REQUESTED** [review]
Resolve the test failures.

**Review by arao6 — COMMENTED** [review]
Build seems to be failing. Please fix the failures mentioned

**Review by arao6 — APPROVED** [review]

**Review by peter-joseph — APPROVED** [review]

**Pull request merged**